docs/reference-guides/slotfills/plugin-pre-publish-panel.md

```diff
@@ -2,11 +2,15 @@
 
 This slot allows for injecting items into the bottom of the pre-publish panel that appears to confirm publishing after the user clicks "Publish".
 
+<div class="callout callout-info">
+`PluginPrePublishPanel` was moved from the `@wordpress/edit-post` package to `@wordpress/editor` in Gutenberg 18.1 (to be included in WordPress 6.6). The deprecated export will be removed with WordPress 6.8.
+</div>
```

Comment:
Not sure if we should keep these callouts. We shouldn't remember to remove them later. 

If we keep them, we should be doing the same for the other slots that moved.